code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
+
+		if("spin")
+			spin(20, 1)
+			message = "<b>[src]</b> spins around dizzily!"
+
+		if("highfive")
+			if(restrained())
+				return
+			if(highfiving)
+				to_chat(src, "You give up on the high-five.")
+				highfiving = FALSE
+				return
+			highfiving = TRUE
+			for(var/mob/living/carbon/C in orange(1))
+				if(C.highfiving)
+					if(C.mind.special_role == "Wizard" && mind.special_role == "Wizard")
+						visible_message("<span class='danger'><b>[name]</b> and </b>[C.name]</b> high-five EPICALLY!</span>")
```

Comment:
Epically is actually a word. It looks wrong and spellcheck doesn't like it, but it is right.

Besides, this is going to be such a rare thing to see anyways, and you probably aren't gonna live through it unless you are one of the wizards involved.